src/services/MuiThemeFactory.tsx

```diff
@@ -57,7 +57,7 @@ export const AppThemeProvider = ({ children }: { children?: React.ReactNode }) =
 
     return (
         <DarkLightModeContext.Provider value={{ darkLightMode, setDarkLightMode }}>
-            <MuiThemeProvider theme={theme}>{children}</MuiThemeProvider>;
+            <MuiThemeProvider theme={theme}>{children}</MuiThemeProvider>
```

Comment:
The culprit is this commit: https://github.com/htbkoo/personal-portfolio/commit/97be0ccfd7b2f27a4a29226073b26413f846c715#diff-5a60df4d813739d14912a4819fe2e3da94e0ce4606ad54b15fda23523fe1de4bR50